<?xml version="1.0"?>

<launch>

  <!-- Run radius outlier removal filter
      - set the radius to search around each point
      - set the min. no. of neighbours to be within the search radius for the point to be included -->
  <node pkg="lidar_snow_removal" type="radiusOutlierFilter" name="radius_outlier_filter" output="screen"/>
    <param name="/radius/inputTopic" value="/velodyne_points" type="string" />
    <param name="/radius/radius_search" value="0.08" type="double" />
    <param name="/radius/minNeighbours" value="3" type="int" />

  <!-- Run rviz -->
  <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lidar_snow_removal)/config/rviz_config.rviz" /> -->

</launch>
